tests/test_blurb.py

```diff
@@ -188,3 +187,68 @@ def test_version(capfd):
     # Assert
     captured = capfd.readouterr()
     assert captured.out.startswith("blurb version ")
+
+
+def test_parse():
+    # Arrange
+    contents = ".. gh-issue: 123456\n.. section: IDLE\nHello world!"
+    blurbs = blurb.Blurbs()
+
+    # Act
+    blurbs.parse(contents)
+
+    # Assert
+    metadata, body = blurbs[0]
+    assert metadata["gh-issue"] == "123456"
+    assert metadata["section"] == "IDLE"
+    assert body == "Hello world!\n"
+
+
+@pytest.mark.parametrize(
+    "contents, expected_error",
+    (
+        (
+            "",
+            r"Blurb 'body' text must not be empty!",
+        ),
+        (
+            "gh-issue: Hello world!",
+            r"Blurb 'body' can't start with 'gh-'!",
+        ),
+        (
+            ".. gh-issue: 1\n.. section: IDLE\nHello world!",
+            r"Invalid gh-issue number: '1' \(must be >= 32426\)",
+        ),
+        (
+            ".. bpo: one-two\n.. section: IDLE\nHello world!",
+            r"Invalid bpo number: 'one-two'",
+        ),
+        (
+            ".. gh-issue: one-two\n.. section: IDLE\nHello world!",
+            r"Invalid GitHub number: 'one-two'",
+        ),
+        (
+            ".. gh-issue: 123456\n.. section: Funky Kong\nHello world!",
+            r"Invalid section 'Funky Kong'!  You must use one of the predefined sections",
```

Comment:
Are you trying to get us sued by Nintendo?!

Reply:
Funky Kong already lives here :D

https://github.com/python/blurb/blob/main/tests/fail/invalid-section.rst?plain=1